apps/notification/serializers.py

```diff
@@ -37,3 +43,58 @@ def get_data(self, notification):
             return notification.data
 
         return {}
+
+
+class AssignmentEntryCommentSerializer(serializers.ModelSerializer):
+    content_id = serializers.IntegerField(source='id', read_only=True)
+    content_title = serializers.CharField(source='text', read_only=True)
+    project = serializers.IntegerField(source='entry.project_id', read_only=True)
+    project_title = serializers.CharField(source='entry.project.title', read_only=True)
```

Comment:
we don't need project data from entry. We can use Assignment Project directly avoiding unnecessary project fetch through the entry.